nilcc-agent/src/config.rs

```diff
@@ -49,7 +50,7 @@ pub struct AgentConfig {
     pub tls: Option<TlsConfig>,
 
     /// The heartbeat verifier configuration.
-    pub heartbeat_verifier: Option<HeartbeatVerifierConfig>,
+    pub verifier_heartbeat: VerifierHeartbeatConfig,
```

Comment:
What does `verifier heartbeat` mean? Who is the verifier here and what do they verify?

Reply:
Are you asking to expand the docs? I'm calling "verifier" the role of `nilav` here.

Reply:
Eventually it will make it to nilav/the verifier, even though it first goes through something else. I wanted something more than "heartbeat" since it's too generic. I also didn't want to call it `nilav` because the name will change (and also I'd like a name more meaningful about "what" it does rather than a code name, whatever it is). What name do you suggest?